ONWServices.Tests/Unit/NewGameTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ONWServices.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ONWServices.Tests.Unit
+{
+    //Game statuses: New, Available, Night Phase, Information Review, Discussion, Round Summary, (Back to Night Phase), Game Summary, Closed
+    //Game must have a minimum of 3 players    
+    //Game can only have 1 Game Master
+
+    [TestClass]
+    public class NewGameTest
```

Comment:
I don't know how you .Net guys keep track of what tests apply to what classes 😄 Would it make sense to call this class just `GameTest`, so the convention would be `<class-being-tested>Test`?